api/src/test/java/org/openmrs/api/db/hibernate/HibernateDiagnosisDAOTest.java

```diff
@@ -92,8 +92,13 @@ public void shouldGetDiagnosisById() {
 	}
 
 	@Test
-	public void shouldGetActiveDiagnoses() {
-		assertEquals(2, diagnosisDAO.getActiveDiagnoses(new Patient(2)).size()); 
+	public void shouldGetActiveDiagnosesWithFromDate() {
```

Comment:
@dkayiwa  Here is the tests that gets diagnosis when supplied with a date

Reply:
This is what i mean. If i supplied two different dates, can i get different results basing on the date value?

Reply:
You can. Let me implement test it out

Reply:
Done